src/traces/scattergl/index.js

```diff
@@ -360,22 +359,26 @@ function clearViewport(comp, vp) {
     gl.clear(gl.COLOR_BUFFER_BIT);
 }
 
+function repeat(opt, cnt) {
+    var opts = new Array(cnt);
+    for(var i = 0; i < cnt; i++) {
+        opts[i] = opt;
+    }
+    return opts;
+}
```

Comment:
non blocking, would probably make a good `Lib` function. Pretty sure we use this in other places as well, though I'm not quite sure how to search for it...

Reply:
done in https://github.com/plotly/plotly.js/pull/2860/commits/33b4085010209af0f37b8a2ca538115be76bfc84

Reply:
I didn't find any other usage of a repeat helper in `src/`, but making a lib function now should help us remember that this helper exists.